### ISSUE TITLE
[feature request] return frontend migrated dashboard JSON

### ISSUE THREAD

**Issue by ryantxu** [grafana/grafana-image-renderer#270]
**What would you like to be added**:

It would be great if we could run the front end dashboard/panel migration scripts and save the the "clean" JSON file.  Given that the process needs to run in a broswer and essentially export the shared

**Why is this needed**:

Currently all "migrations" happen on load in the browser -- ideally, we would like to save JSON models with valid schema so that external tools can work consistently as well.  In 8.2, we aim to have an accurate cue schema for dashboards, and be able to validate on the server.  It would be great to be able to script downloading all dashboards after running though frontend migrations.

**Comment by macabu** [comment]
@ryantxu I dont think this is applicable anymore, seems to be related to an old feature toggle around Angular migration? Let me know in case this issue is still relevant and we can reopen!
